avro/src/types.rs

```diff
@@ -2771,35 +2771,36 @@ Field with name '"b"' is not a member of the map items"#,
         use crate::ser::Serializer;
         use serde::Serialize;
 
-        let schema_str = r#"{
+        let schema_str = r#"
+        {
             "type": "record",
             "name": "NamespacedMessage",
-            [NAMESPACE]
```

Comment:
Why is this removed ?

Reply:
Oh my bad, it took me some time to fix this schema, and then I forgot to add the namespace again, I'll update it